extensions/hashicorp-vault/README.md

```diff
@@ -0,0 +1,9 @@
+# [HashiCorp Vault](https://www.vaultproject.io/) Extension
+
+## Configuration
```

Comment:
Please add a section that explains how to run a local vault instance to run the integration tests